File: Task-3/python/SubTask-3.py
x = int(input("Enter the number of lines for the upper half of the diamond: "))

if x > 0:
    for i in range(x):
        print(' ' * (x - i - 1) + '*' * (i * 2 + 1))
    
    for i in range(x - 2, -1, -1):
        print(' ' * (x - i - 1) + '*' * (i * 2 + 1))
else:
    print("Please enter a positive integer.")

File: Task-3/Javascript/SubTask-3.js
const n = parseInt(prompt('Enter the number of lines for the diamond:'), 10);
let diamond = '';

if (Number.isInteger(n) && n > 0) {
    for (let i = 0; i < n; i++) {
        diamond += ' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1) + '\n';
    }
    for (let i = n - 2; i >= 0; i--) {
        diamond += ' '.repeat(n - i - 1) + '*'.repeat(2 * i + 1) + '\n';
    }
    console.log(diamond);
} else {
    console.log('Please enter a positive integer.');
}
